h2/test_h2_specs.py

```diff
@@ -74,7 +74,7 @@ class H2Spec(tester.TempestaTest):
             'type' : 'external',
```

Comment:
Please update the copyright year in the file and other files, which you ever touch.